src/query/Index.hpp

```diff
@@ -86,6 +86,7 @@ class Index {
    */
   ProjectionMatch findNearestProjection(const Eigen::VectorXd &location, int n);
 
+  ProjectionMatch findNearestVolume(const Eigen::VectorXd &location, int n);
```

Comment:
Maybe we can come up with a better name for this.
This will look for the cell including the point and return the nearest vertex as a fallback.